### PR TITLE
Clean amalgamation ws and skip failing test

### PR DESCRIPTION
**Pull request by sandeep-krishnamurthy**
@madjam

### CODE REVIEW

File: tests/python/unittest/test_loss.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ def get_net(num_hidden):
 
 
 def test_ce_loss():
+    # Skipping this test to have nightly build passing.
+    # There seems to be an issue and tracked at - issue 7677
+    return
```
Comment (szha) [inline tests/python/unittest/test_loss.py]: This is not how it should be done. Aston is fixing loss problems in #7659.

How frequent is this test failing.

Comment (madjam) [reply tests/python/unittest/test_loss.py]: Roughly 15%. I do hear your concern but it doesn't look like there is a risk of this issue getting dropped. Do you have a better suggestion? Unreliable builds and flaky tests are hurting us in other ways.

Comment (szha) [reply tests/python/unittest/test_loss.py]: Drop the change on test_ce_loss test in this PR and work with @astonzhang to fix this test in his PR.

Comment (astonzhang) [reply tests/python/unittest/test_loss.py]: #7659 proposed some change in the unittest

Comment (madjam) [reply tests/python/unittest/test_loss.py]: @sandeep-krishnamurthy lets get @astonzhang 's PR in and keep this only for amalgamation test.

Comment (astonzhang) [reply tests/python/unittest/test_loss.py]: The unittest fix is now available in a separate pr #7693

**Comment by sandeep-krishnamurthy** [comment]
Remove logic to skipping the failing test case. Thanks to Aston for fixing it.
This PR now contains only fix for amalgamation min CI.
@madjam - Can we merge this?

**Pull request merged**